pkg/authorization/registry/util/convert.go

```diff
@@ -0,0 +1,138 @@
+package util
+
+import (
+	"k8s.io/apimachinery/pkg/conversion"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
+)
+
+// ClusterRoleToRBAC turns an OpenShift ClusterRole into a Kubernetes RBAC
+// ClusterRole, the returned object is safe to mutate
+func ClusterRoleToRBAC(obj *authorizationapi.ClusterRole) (*rbac.ClusterRole, error) {
```

Comment:
these are specific to the registry and used only here, as they do a deep copy.
Ideally these conversion will survive only for the legacy endpoints eventually.